common/alloc.c

```diff
@@ -7086,6 +7086,7 @@ alloc_event_bb_insert(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
     app_pc pc = instr_get_app_pc(inst);
     if (pc == NULL)
         return DR_EMIT_DEFAULT;
+    drmgr_disable_auto_predication(drcontext, bb);
```

Comment:
Including a comment explaining why we're disabling and whether it's meant to be temporary (plus the issue # if temporary) will improve code readability.

Reply:
Done.